static/css/components/list-showcase.less

```diff
@@ -3,7 +3,7 @@
 @import (less) "components/book.less";
 
 @my-lists-item-border: black;
-.my-lists {
```

Comment:
I think I'd just change this to `.my-lists .showcase` -- i.e. show these rules if both styles are present

Reply:
That's the ticket!